### ISSUE TITLE
Chinese language pack

### ISSUE THREAD

**Issue by ignatov** [pingfangx/TranslatorX#65]
Hi, I'm a JetBrains employee, and we're working on initial support for the Chinese language pack. If you're interested in, feel free to contact me: ignatov@jetbrains.com

Cheers, Sergey.

**Comment by ghost** [comment]
good news

**Comment by chinkiver** [comment]
very good news

**Comment by whcoding** [comment]
very good news

**Comment by KevinZonda** [comment]
I love it!

**Comment by icaohui** [comment]
nice

**Comment by getda** [comment]
期待中文版的到来哦，目前使用的Sublime

**Comment by pingfangx** [comment]
好消息：官方中文插件在 2020.1 的 EAP 版本中已经有啦，估计正式版都会有的。
[Chinese ​(Simplified)​ Language Pack](https://plugins.jetbrains.com/plugin/13710-chinese-simplified-language-pack)
作者之前向楼主的邮箱回复了邮件，但没有收到进一步回复，可能是因为我英语比较烂。
不知道官方汉化包与该项目是否有部分关联，我看了下可能是有的。
不过这不重要，初衷也是让大家用上汉化的软件，以后有了官方汉化包，大家就更方便啦，可喜可贺，撒花~

**Comment by ignatov** [comment]
@pingfangx You're right, that's the plugin that we'd like to ship with 2020.1. I've just returned from vacation and will reply to you soon.

**Comment by 18595476026** [comment]
very good！！！

**Comment by WPTomo** [comment]
nice!

**Comment by vincent66666** [comment]
期待

**Comment by NezhaGrin** [comment]
nice!

**Comment by SekiBetu** [comment]
哇，海盗转正了，恭喜

**Comment by mustangten** [comment]
Very look forward to this good news~

**Comment by BinZhiZhu** [comment]
how to use
